src/engraving/dom/line.cpp

```diff
@@ -582,11 +583,14 @@ LineSegment* LineSegment::rebaseAnchor(Grip grip, Segment* newSeg)
         anchorChanged = true;
     }
 
-    if (newSeg->system() != oldSystem) {
+    const Segment* newSpannerSeg = left ? l->startSegment() : l->endSegment();
+    const System* newSpannerSystem = newSpannerSeg->measure()->system();
```

Comment:
All good, but naming this newSpannerSeg confused the hell out of me at first read as my head immediately thought it was a SpannerSegment* (as opposed to a Segment*).